product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Datadog"
+---
+
+Datadog integration is based on the OpenTelemetry Collector and the Datadog plugin bundled with the OpenTelemetry collector. Once enabled, BigAnimal automatically configures data planes to send the Telemetry to your Datadog account via Datadog's cloud receive endpoint.
+
+## Pre-requisites
+
+The Datadog account is required to enable this integration.
+
+## Enabling Datadog
+
+To enable the **Datadog** integrations: 
+
+1. Select an existing project from the **Projects** page.
+2. Go to **Settings** on the left-side navigation.
+3. Select **Integrations** from the **Settings** drop down list.
+4. Select **Datadog**, a window pops-up to provide the details: 
+   - **Datadog API Key** &mdash; Provide an API Key. For more details, see [Datadog API Key](https://docs.datadoghq.com/account_management/api-app-keys/)
```

Comment:
Should we write in the doc that his field is considered "sensitive" so it will not be shown once set when editing an existing configuration? I mean, it can only be set but when you edit the configuration later you are not able to see the value that you added before.